apps/mark-scan/frontend/src/pages/casting_ballot_page.tsx

```diff
@@ -1,13 +1,10 @@
-import { H1, Main, Text, Screen, ProgressBar } from '@votingworks/ui';
+import { Main, Screen, Loading } from '@votingworks/ui';
 
 export function CastingBallotPage(): JSX.Element {
   return (
     <Screen white>
       <Main centerChild padded>
-        <Text center>
-          <ProgressBar duration={4000} />
-          <H1>Casting Your Official Ballot...</H1>
-        </Text>
```

Comment:
I agree the ellipsis looks weird. It seems unnecessary to animate the ellipsis since there's already another animation happening. I think one animation at a time is a good visual rule of thumb.